modules/core/src/lib/attribute/gl-utils.ts

```diff
@@ -62,11 +62,15 @@ export function getBufferAttributeLayout(
   };
 }
 
+export function getStride(accessor: DataColumnSettings<any>): number {
+  return accessor.stride || accessor.size * accessor.bytesPerElement;
+}
+
 export function bufferLayoutEqual(accessor1: BufferAccessor, accessor2: BufferAccessor) {
   return (
     (accessor1.type ?? GL.FLOAT) === (accessor2.type ?? GL.FLOAT) &&
     accessor1.size === accessor2.size &&
-    accessor1.stride === accessor2.stride &&
+    getStride(accessor1) === getStride(accessor2) &&
```

Comment:
This does not look correct to me. `BufferAccessor` does not contain a `bytesPerElement`. The right solution could be always passing `Required<BufferAccessor>` into this function.

Reply:
Updated types to match what is being used in `attribute/attribute.ts`